### PR TITLE
Make the Website Responsive

### PR DESCRIPTION
**Pull request by alanluqman**
## Description 
In this Pull Request,  I modified index.html and  style.css properties to make the Website responsive to Desktop.

**Files changed**

- index.html
- style.css

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -317,52 +451,118 @@ body {
...
+  .work-card {
+    width: 100%;
+  }
+
+  .card-button:hover {
+    color: var(--color-sccent);
+    background-color: var(--color-secondary);
+    border: 1px solid var(--color-sccent);
+  }
+}
```
Comment (leonardodiasb) [inline style/style.css:509]: - [ ] By setting the min-width of 768px of your media query, part of this section is cut out of the screen. Kindly fix the margin and padding of this section.
- [ ] The image width of the featured project should take a more significant portion of the screen size. Kindly fix that.
image
image

File: style/style.css [hunk 2]
```diff
@@ -317,52 +451,118 @@ body {
...
+
+  .work-card {
+    width: 100%;
+  }
+
+  .card-button:hover {
+    color: var(--color-sccent);
+    background-color: var(--color-secondary);
+    border: 1px solid var(--color-sccent);
+  }
```
Comment (leonardodiasb) [inline style/style.css:508]: - [ ] "See Project" button should appear only on hover. Kindly fix that.
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @alanluqman, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️
Your CSS style is amazing. ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Kindly set the media queries after all your CSS styles instead of opening multiple media queries through the document.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @alanluqman , 

Great job following up with the changes requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved ✔️

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**